app/page.tsx

```diff
@@ -1,3 +1,25 @@
-export default function Main() {
-    return <h1>Root page</h1>
+import Header from './components/header'
+import ListItem from './components/list-item'
+import { ListItemType } from './app-types'
+
+const listItems: ListItemType[] = [
```

Comment:
Instead of having static data here, we can make use of any mock API. 
[https://www.mockable.io/](https://www.mockable.io/)
[https://designer.mocky.io/](https://designer.mocky.io/)

Reply:
I'll update in my further implementation. Changes are in-progress for it.